content/docs/run-api-server/installing.mdx

```diff
@@ -21,69 +18,59 @@ SDF publishes new releases to its custom Ubuntu repositories. Follow [this guide
 <CodeExample>
 
 ```bash
-sudo apt update && sudo apt install stellar-captive-core stellar-horizon
+sudo apt update && sudo apt install stellar-core stellar-horizon
 ```
 
 </CodeExample>
 
-[This page](https://github.com/stellar/packages/blob/master/docs/installing-individual-packages.md#installing-individual-packages) outlines the various commands that these packages make available.
+If you want to do [ingestion](./running.mdx#ingesting-transactions) on this machine, you should also install the `stellar-captive-core` package.  [This page](https://github.com/stellar/packages/blob/master/docs/installing-individual-packages.md#installing-individual-packages) outlines the various commands that these packages make available.
```

Comment:
`stellar-captive-core` is a configuration package. It configures core qorumset for use with captive core for convenience but it's not required. Some operators might choose to configure core themselves using config management, if they choose to do so it might even be better to avoid this package. @brahman81 should be able to confirm this.

Reply:
Also the link this paragraph points at does not contain information about stellar-captive-core

Reply:
Ah, very interesting, thanks for the clarification! The link was intended to point to `stellar-core-cmd` and `stellar-horizon-cmd`, but I can see how that would be confusing. I've moved some things around in 0d944ff.

Reply:
Thanks, it looks great from my perspective now!